src/MissionManager/SurveyMissionItem.cc

```diff
@@ -276,6 +281,7 @@ bool SurveyMissionItem::load(const QJsonObject& complexObject, QString& errorStr
     _gridAltitudeFact.setRawValue           (complexObject[_jsonGridAltitudeKey].toDouble());
     _gridAngleFact.setRawValue              (complexObject[_jsonGridAngleKey].toDouble());
     _gridSpacingFact.setRawValue            (complexObject[_jsonGridSpacingKey].toDouble());
+    _turnaroundDistFact.setRawValue         (complexObject[_jsonTurnaroundDistKey].toDouble());
```

Comment:
How does Qt react when this key is missing from the json? Does it pop any qWarnings? Does it return 0.0 for the value?